pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -689,6 +700,31 @@ public int getSegmentRelocatorFrequencyInSeconds() {
         });
   }
 
+  public boolean getSegmentRelocatorRebalanceConfigBestEfforts() {
```

Comment:
Same for other method names to match the config
```suggestion
  public boolean getSegmentRelocatorBestEfforts() {
```